rosdep/python.yaml

```
@@ -6493,6 +6493,12 @@ python3-hypothesis:
   fedora: [python3-hypothesis]
   gentoo: [dev-python/hypothesis]
   ubuntu: [python3-hypothesis]
+python3-icecream:
+  debian:
+    bookworm: [python3-icecream]
```

This is available for all current release of Debian, not just `bookworm`.

```suggestion
  debian: [python3-icecream]
```